src/main/java/seedu/duke/data/ShoppingList.java

```diff
@@ -1,17 +1,12 @@
 package seedu.duke.data;
 
-import seedu.duke.commands.Command;
-import seedu.duke.commands.IncorrectCommand;
-import seedu.duke.commands.MarkCommand;
-import seedu.duke.commands.UnmarkCommand;
 import seedu.duke.ui.CommandLineTable;
 
 import java.util.ArrayList;
 
 public class ShoppingList {
 
     private static ArrayList<Item> items = new ArrayList<>();
-    private static Command newCommand;
```

Comment:
Thanks for doing some housekeeping for this class! :+1:

Reply:
No problem!